docs/source/60_building_apps.md

```diff
@@ -103,15 +109,16 @@ ADAM ACTIONS
 $ ./bin/adam-submit myCommand input.foo
 ```
 
-Then consider making a pull request to include the new command in ADAM!
+Then consider making a
+[pull request](https://github.com/bigdatagenomics/adam/pulls) to include the
```

Comment:
We have a detailed CONTRIBUTING.md with info for people contributing. Just add a heads up that they should float the issue first, because the way we've written it here makes it sound like we'll merge any and all PRs against ADAM. We've tryed to bias towards accepting contributions, but we can't accept everything, and we're probably more hesitant to accept CLI additions relative to API additions.